int32 lane_id
int32 laneno
int32 navgroad1
bool isPositive1
int32 navgroad2
bool isPositive2
string seqner
